packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -111,7 +115,7 @@ export class TruncatedFormat extends React.Component<ITruncatedFormatProps, ITru
                     <div className={Classes.TABLE_TRUNCATED_VALUE} ref={this.handleContentDivRef}>{cellContent}</div>
                     <Popover
                         className={Classes.TABLE_TRUNCATED_POPOVER_TARGET}
-                        constraints={constraints}
+                        tetherOptions={tetherOptions}
```

Comment:
how about `tetherOptions={{ constraints }}`? let the typechecker do the work. then you can revert the change above.